src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    private String name;
+
+    public Excavator(String name) {

```

Comment:
Remove constructor and "name" field to be able to create instances without providing a name